packages/api/README.md

```diff
@@ -136,3 +136,55 @@ We use [pickup](https://github.com/web3-storage/pickup) to fetch DAGs from IPFS
 - `PICKUP_BASIC_AUTH_TOKEN` must be set as a secret in the env.
 
 For local dev, we use a local ipfs-cluster container for the same service.
+
+## w3up
+
+Some uploads sent to nft.storage/api will be sent to up.web3.storage (aka 'w3up') for storage, serving on IPFS, and persistence to filecoin.
+
+All uploads sent to w3up will be stored in the same web3.storage space configured by env var `W3_NFTSTORAGE_SPACE`.
+
+### using console.web3.storage to browse uploads to w3up
+
+You can use console.web3.storage to browse uploads in the W3_NFTSTORAGE_SPACE.
+
+The DID used by your console.web3.storage session will need to be authorized to access the space.
+
+The credentials used in staging/production are in the usual vault of secrets under 'w3up credentials'.
+
+Run the cli command `w3up console ucan generate`
```

Comment:
maybe better to use the yarn command? 
```suggestion
Run the cli command `yarn w3up:console:ucan:generate`
```

Reply:
by 'the cli command' I meant 'arguments passed to the ./scripts/cli.js I found'. So I think the way I wrote it was ambiguous and probably should change, but not necessarily to one that mentions yarn.